backend/src/tracking/camera.py

```diff
@@ -0,0 +1,74 @@
+"""Camera module implements the camera connection and person detection."""
+
+import cv2
+from numpy import ndarray
+from .people_detector import PeopleDetector
+
+
+class Camera:
+    """The Camera class continuously reads frames from the given camera and performs
+    feature detection on them.
+
+    :param int cameraId: Index of the camera to use
+    """
+
+    def __init__(self, cameraId: int = 0):
+        self.exiting = False
+        self.capture = cv2.VideoCapture(cameraId)
+        self.detector = PeopleDetector()
+        self.on_frame = None
+
+        if self.capture.isOpened() is not True:
+            raise RuntimeError('Camera with id ' +
+                               str(cameraId) + ' is not available')
+
+    def stop(self) -> None:
+        """Stops reading from the camera after the current frame has been processed."""
+        self.exiting = True
+
+    def process(self) -> None:
+        """Processes the camera frames until `stop()` has been called
+        or the camera is no longer available."""
+        try:
+            while self.exiting is False and self.capture.isOpened() is True:
+                _, frame = self.capture.read()
+                resized_frame = self.resize(frame, 500)
```

Comment:
Maybe move the `500` magic value to a constant so it's more obvious what can be configured?

Reply:
Yes, that makes sense. I will change it 👍